Gemfile

```diff
@@ -67,3 +67,5 @@ group :test do
   gem 'shoulda-matchers', '~> 5.0'
   gem 'webdrivers'
 end
+
+gem 'validate_email', '~> 0.1.6'
```

Comment:
Why the gem? I think it would be clearer if we put the regex on the model instead - that way, we can make any changes to it if ever necessary